pyproject.toml

```diff
@@ -1,21 +1,33 @@
 [build-system]
-requires = [
-  # sync with setup.py until we discard non-pep-517/518
-  "setuptools>=45.0",
-  "setuptools-scm[toml]>=6.2.3",
-  "wheel",
-]
-build-backend = "setuptools.build_meta"
-
-[tool.setuptools_scm]
-write_to = "pytest_metadata/__version.py"
+ requires = ["poetry-core>=1.0.0"]
+ build-backend = "poetry.core.masonry.api"
 
 [tool.poetry]
 name = "pytest-metadata"
-version = "2.0.0"
+version = "2.0.1-rc0"
 description = "pytest plugin for test session metadata"
-authors = ["Dave Hunt"]
 license = "Mozilla Public License 2.0 (MPL 2.0)"
```

Comment:
```suggestion
license = "MPL-2.0"
```

Reply:
I was just going to ask you about that, haha.